### DISCUSSION TITLE
How do I change the display name?

### DISCUSSION THREAD

**Discussion by dustbro** [the-djmaze/snappymail#1021] [Q&A] [answered]
I'm using SnappyMail with Nextcloud. How do I change the name displayed to email recipients? 
For example, my email is myname@mydomain.com. The recipient sees the name as "myname". All lower case... no spaces. 
I can't seem to find the place to change this is nextcloud.

**Comment by cm-schl** [comment, marked as answer]
If it's only you that would like to change the name (not a bunch of users you would like to change as an admin) you have to enter in your personal settings (lower left in SnappyMail), then go to "General" and click on your mail address beside the caption "Identity". There you can change the name that is used when sending mails.

**Reply by dustbro** [reply]
Thanks so much! This worked.
